test/jobs.ServiceInstanceUpdateJob.spec.js

```diff
@@ -236,10 +236,10 @@ describe('Jobs', function () {
         organization_guid: organization_guid
       });
       const diff = [
-        ['jobs:', null],
-        ['- name: blueprint_z1', null],
-        ['  instances: 2', 'removed'],
-        ['  instances: 1', 'added']
+        ['instance_groups:', null],
```

Comment:
ADD UT for the new section of Code written. Exclusively test with all the changes ex., instance group, persitence disk, job removal.

Reply:
ok.. will add another test case which tests the job removal. Persistence disk type is exactly the same as instances, the call flow is same too. So, adding a test case for job removal which will test the newly added functions.